services/src/main/java/com/linkedin/kmf/services/configs/ProduceServiceConfig.java

```diff
@@ -57,16 +59,19 @@
   public static final String REBALANCE_ENABLED_DOC = "Periodically move leaders and replica assignments so they are "
       + "evenly spread amongst brokers.";
 
-  public static final String REBALANCE_PARTITION_MULTIPLE_CONFIG = "produce.topic.rebalancePartitionMultiple";
-  public static final String REBALANCE_PARTITION_MULTIPLE_DOC = "Determines the number of partitions per broker in the ideal case.";
+  public static final String REBALANCE_PARTITION_FACTOR_CONFIG = "produce.topic.rebalancePartitionFactor";
+  public static final String REBALANCE_PARTITION_FACTOR_DOC = "Determines the number of partitions per broker in the ideal case.";
```

Comment:
I find it a bit hard to interpret "ideal" here. This is actually the default number of partitions when we create/expand the partitions of a topic, right? Maybe we can say something like "The default ratio of partitions to broker when RebalanceService create/expand partitions of a topic".

I am also thinking that we can allow the config name to explain itself better -- how about "rebalance.default.partitionToBrokerRatio"